packages/terra-embedded-component/examples/Index.site-page.jsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
```

Comment:
Moved from `terra-framework-site/src/examples/embedded-component` & update to use the `IndexPageTemplate`